include/deal.II/grid/grid_in.h

```diff
@@ -948,6 +967,17 @@ class GridIn
    * Input format used by read() if no format is given.
    */
   Format default_format;
+
+  /**
+   * Data member that stores field data defined at the cells of the mesh.
+   * The format is as follows:
+   * - std::string stores the name of the field data (identifier) as specified
+   * in the external mesh
+   * - std::vector<double> stores value for the given identifier in each
+   * cell_id.
+   * To access the value use field_data[name_field][cell_id].
+   */
+  std::map<std::string, std::vector<double>> field_data;
```

Comment:
```suggestion
  std::map<std::string, Vector<double>> cell_data;
```
same documentation updates as above